src/utils/Logger.ts

```diff
@@ -0,0 +1,30 @@
+import { LogInterface } from "../interfaces/LogInterface";
+
+export class Logger implements LogInterface {
+  public debug(msg: string, ...supportingData: any[]): void {
+    this.emitLogMessage("debug", msg, supportingData);
+  }
+  public info(msg: string, ...supportingData: any[]): void {
```

Comment:
We don't need to define the return type as it's implicitly defined which is usually preferred

Reply:
Still a few methods with this return type.